src/renderers/rust/templates/instructionsPage.njk

```diff
@@ -0,0 +1,117 @@
+{% extends "layout.njk" %}
+{% import "macros.njk" as macros %}
+
+{% block main %}
+
+{{ imports }}
+
+/// Accounts.
+pub struct {{ instruction.name | pascalCase }} {
+{% for account in instruction.accounts %}
+  {% if account.docs.length > 0 %}
+    {{ macros.docblock(account.docs) }}
+  {% endif %}
+
+  {% if account.isSigner === 'either' %}
+    {% set type = '(solana_program::pubkey::Pubkey, bool)' %}
+  {% else %}
+    {% set type = 'solana_program::pubkey::Pubkey' %}
+  {% endif %}
+
+  {% if account.isOptional %}
+    pub {{ account.name | snakeCase }}: Option<{{ type }}>,
+  {% else %}
+    pub {{ account.name | snakeCase }}: {{ type }},
+  {% endif %}
+{% endfor %}
+}
+
+impl {{ instruction.name | pascalCase }} {
+  pub fn instruction(&self{{ ', args: ' + instruction.name | pascalCase + 'InstructionArgs' if hasArgs }}) -> solana_program::instruction::Instruction {
+    {% if hasArgs === false %}
+    let args = {{ instruction.name | pascalCase }}InstructionArgs::new();
+    {% endif %}
+    solana_program::instruction::Instruction {
+      program_id: crate::{{ program.name | snakeCase | upper }}_ID,
```

Comment:
```suggestion
      program_id: crate::generated::{{ program.name | snakeCase | upper }}_ID,
```

alternatively, if we don't want to make any assumptions about the library structure above the `generated/` folder,

```suggestion
      program_id: super::super::{{ program.name | snakeCase | upper }}_ID,
```